pkg/controller/workspace/workspace_controller.go

```diff
@@ -212,6 +204,15 @@ func (r *ReconcileWorkspace) Reconcile(request reconcile.Request) (reconcileResu
 		componentDescriptions = append(componentDescriptions, cheRestApisComponent)
 	}
 
+	if err := provision.SyncPVC(workspace, componentDescriptions, r.client, reqLogger); err != nil {
+		return reconcile.Result{}, err
+	}
+
+	//TODO Previously we check rbac before components creation. Now we are able to do the same but what the purpose of creating RBAC if like there is component with unreached plugin/editor?
```

Comment:
1. Just remove this TODO or 2. move to old place.
+1 for 1 option but up to you